components/public-api-server/pkg/apiv1/oidc.go

```diff
@@ -118,7 +130,19 @@ func (s *OIDCService) ListClientConfigs(ctx context.Context, req *connect.Reques
 		return nil, err
 	}
 
-	return nil, connect.NewError(connect.CodeUnimplemented, errors.New("gitpod.experimental.v1.OIDCService.ListClientConfigs is not implemented"))
+	configs, err := db.ListOIDCClientConfigsForOrganization(ctx, s.dbConn, organizationID)
```

Comment:
Here, we list the configs directly in Public API, rather than delegating to AIM. This is in light of https://github.com/gitpod-io/gitpod/issues/15860